libsql/internal/ws/driver.go

```diff
@@ -86,21 +86,29 @@ func convertToNamed(args []driver.Value) []driver.NamedValue {
 }
 
 func (s stmt) Exec(args []driver.Value) (driver.Result, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), defaultWSTimeout)
-	defer cancel()
-	return s.c.ExecContext(ctx, s.query, convertToNamed(args))
+	return s.c.ExecContext(context.Background(), s.query, convertToNamed(args))
```

Comment:
Why it's ok to remove the timeout?

Reply:
Good question. So, earlier we didn't have ctx methods and users had to use non-ctx methods. Now, they can use either. So, if they want timeouts, they can use ctx methods. 

Second, adding a timeout in this place is tricky anyways. If we keep it small, then large queries might get cancelled pre emptively. The user probably has a better idea of time required for the query. This is how most other client libraries in Go behave.

Reply:
Good point.